Source/Protocols/Request.swift

```diff
@@ -12,7 +11,8 @@ public protocol Request {
 
 public extension Request where ResponseObject: Decodable {
   func parse(_ data: Data) throws -> ResponseObject {
-    return try JSONDecoder().decode(ResponseObject.self, from: data)
+    let decoder = JSONDecoder()
+    return try decoder.decode(ResponseObject.self, from: data)
```

Comment:
Would you mind inlining the `decoder` variable so that this no longer shows up in the diff?